hack/scripts/integration-test-kind.sh

```diff
@@ -3,7 +3,7 @@
 set -o errexit
 set -o nounset
 
-KUBERNETES_VERSION=v${KUBERNETES_VERSION:-1.13.6}
+KUBERNETES_VERSION=v${KUBERNETES_VERSION:-1.13.10}
```

Comment:
```suggestion
KUBERNETES_VERSION=v${KUBERNETES_VERSION:-1.13.11}
```

Reply:
Kind does not support `1.13.11`, the latest `v1.13` is `v1.13.10`.